rag/prompt.py

```
@@ -0,0 +1,35 @@
+"""Prompt file management utilities."""
+
+from pathlib import Path
+import os
+import threading
+from watchfiles import watch
```

Pylint (3.3.7)

[error] Unable to import 'watchfiles'

(E0401)